huaweicloud/services/gaussdb/transport.go

```diff
@@ -0,0 +1,207 @@
+package gaussdb
```

Comment:
I see that some functions are used in some other resources or data sources in huaweicloud pakcage, so suggest put the `transport.go` file the common path.